app/shop.py

```diff
@@ -0,0 +1,50 @@
+from app.customer import Customer
+from math import sqrt
+import datetime
+
+
+class Shop:
+    def __init__(self, name: str, location: list, products: dict) -> None:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]